projects/objects/freight/protos/IntermodalOfficeContainer.proto

```diff
@@ -0,0 +1,120 @@
+#VRML_SIM R2019a utf8
+# license: Copyright Cyberbotics Ltd. Licensed for use only with Webots.
+# license url: https://cyberbotics.com/webots_assets_license
+# An intermodal container is a large standardized shipping container, designed and built for intermodal freight transport, meaning these containers can be used across different modes of transport – from ship to rail to truck – without unloading and reloading their cargo.
+# This container contains an office.
```

Comment:
I would avoid copy/pasting descriptions (this is painful to read in the user guide).
```suggestion
# An intermodal container with an office room.
```